refinery/data_set_manager/tests.py

```diff
@@ -1281,6 +1289,53 @@ def test_update_attribute_order_ranks_invalid(self):
         attribute_list = AttributeOrder.objects.filter(assay=self.assay)
         self.assertItemsEqual(old_attribute_list, attribute_list)
 
+    def test_get_file_url_from_node_uuid(self):
+        test_file_a = StringIO()
+        test_file_a.write('Coffee is great.\n')
+        file_store_item_a = FileStoreItem.objects.create(
+            datafile=InMemoryUploadedFile(
```

Comment:
Perhaps a better alternative is to delete all the FileStoreItems in a tearDown method explicitly and/or use [SimpleUploadedFile](http://stackoverflow.com/a/20508621) or [mock the File instance](http://stackoverflow.com/a/4547203).